docs/api/browser-window.md

```diff
@@ -498,6 +498,14 @@ Returns:
 
 Emitted on 3-finger swipe. Possible directions are `up`, `right`, `down`, `left`.
 
+#### Event: 'begin-sheet' _macOS_
```

Comment:
What do you think about calling these `sheet-begin` and `sheet-end` instead?

Then they would follow the naming pattern of the `scroll-touch-*` events.